### PR TITLE
Fix for customer ticket

### PR DESCRIPTION
**Pull request by archanaponnada**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000
**Fix for issue** : https://github.ibm.com/project-fortress/pm/issues/4185
Output from acceptance testing:
image
image

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/service/scc/data_source_ibm_scc_posture_collectors.go [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+
+	userDetails, err := meta.(conns.ClientSession).BluemixUserDetails()
+	if err != nil {
+		return diag.FromErr(fmt.Errorf("[ERROR] Error getting userDetails %s", err))
+	}
+
+	accountID := userDetails.UserAccount
+	listCollectorsOptions.SetAccountID(accountID)
+
+	collectorList, response, err := postureManagementClient.ListCollectorsWithContext(context, listCollectorsOptions)
```
Comment (hkantare) [inline ibm/service/scc/data_source_ibm_scc_posture_collectors.go:285]: This ListCollectorsWithContext handles pagination? If we have more than 50 or 100 will List give all or only first page?
If not so you need to add pagination  support

File: ibm/service/scc/data_source_ibm_scc_posture_credentials.go [hunk 2]
```diff
@@ -0,0 +1,571 @@
...
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "AWS arn value.",
+									},
+									"username": &schema.Schema{
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "username of the user.This is mandatory for DataBase, Kerbros,OpenStack Credentials.",
+									},
+									"password": &schema.Schema{
```
Comment (hkantare) [inline ibm/service/scc/data_source_ibm_scc_posture_credentials.go:137]: Can we add Sensitive: true to those which are some sensitive information like passowrd , apikey, seceretid...so on
Just check which are all can be sensitive

File: website/docs/d/scc_posture_collectors.html.markdown [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+* `id` - The unique identifier of the list_collectors.
+* `collectors` - (Required, List) The array of items returned.
+Nested scheme for **collectors**:
+	* `approved_internet_gateway_ip` - (Optional, String) The approved internet gateway ip of the collector. This field will be populated only when collector is installed.
```
Comment (hkantare) [inline website/docs/d/scc_posture_collectors.html.markdown:28]: In computed attributes we can have required/optional.. Remove from all 
`approved_internet_gateway_ip` - (String)

File: ibm/service/scc/data_source_ibm_scc_posture_collectors.go (hunk 1 above)
Comment (archanaponnada) [reply ibm/service/scc/data_source_ibm_scc_posture_collectors.go:285]: We have pagination support. If no limit is given, it pulls only first page

File: website/docs/d/scc_posture_collectors.html.markdown (hunk 3 above)
Comment (archanaponnada) [reply website/docs/d/scc_posture_collectors.html.markdown]: Code modified as per review comments

File: ibm/service/scc/data_source_ibm_scc_posture_credentials.go (hunk 2 above)
Comment (archanaponnada) [reply ibm/service/scc/data_source_ibm_scc_posture_credentials.go:139]: Code modified as per review comments

**Pull request merged**